src/index.ts

```diff
@@ -2,9 +2,9 @@
 // Node module: loopback-next-hello-world
 // This file is licensed under the MIT License.
 // License text available at https://opensource.org/licenses/MIT
-import {CoreBindings} from '@loopback/core';
-import {HelloWorldApp} from './application';
-export {HelloWorldApp};
+import { CoreBindings } from '@loopback/core';
+import { HelloWorldApp } from './application';
+export { HelloWorldApp };
```

Comment:
coding style - no spaces inside/around curly braces

you can simply revert this file